README.md

```diff
@@ -59,4 +53,115 @@ Run following commands to proceed with the installation:
   openebs-ndm-tgpmk                              1/1     Running   0          2m1s
   openebs-provisioner-cd5759f96-jfcxb            1/1     Running   0          2m3s
   openebs-snapshot-operator-5f87bd54bf-mmtlh     2/2     Running   0          2m2s
+  openebs-jiva-csi-controller-0                  4/4     Running   0          6m14s
+  openebs-jiva-csi-node-56t5g                    2/2     Running   0          6m13s
+  openebs-jiva-csi-node-xtyhu                    2/2     Running   0          6m20s
+  openebs-jiva-csi-node-h2unk                    2/2     Running   0          6m38s
   ```
+### Steps to provision a Jiva Volume
+
+1. Create Jiva volume policy to set various policies for creating
+   jiva volume.
+   A sample jiva volume policy CR looks like:
+   ```
+    apiVersion: openebs.io/v1alpha1
+    kind: JivaVolumePolicy
+    metadata:
+      name: example-jivavolumepolicy
+      namespace: openebs
+    spec:
+      replicaSC: openebs-hostpath
+      enableBufio: false
+      autoScaling: false
+      target:
+        replicationFactor: 1
+        # monitor: false
+        # auxResources:
+        # tolerations:
+        # resources:
+        # affinity:
+        # nodeSelector:
+        # priorityClassName:
```

Comment:
IMO we shall add some examples related to each policy feature listed in policy Spec for better understanding.